libs/zephyr-edge-contract/src/index.ts

```diff
@@ -38,8 +41,39 @@ export type {
 export {
   ZEPHYR_MANIFEST_FILENAME,
   ZEPHYR_MANIFEST_VERSION,
+  type OTAManifest,
   type ZephyrManifest,
 } from './lib/zephyr-manifest';
+export type {
+  CreateZephyrRuntimePluginWithOTA,
+  ManifestCache,
+  ManifestCacheEntry,
+  ZephyrRemoteUrlChangeDetail,
+  ZephyrRemoteUrlChangeEvent,
+  ZephyrRuntimePluginInstance,
+  ZephyrRuntimePluginOTAOptions,
+} from './lib/ota-runtime-types';
+
+// validation schemas and validators
+export {
+  BundleMetadataSchema,
+  ZephyrDependencySchema,
+  ZephyrManifestSchema,
+  OTAManifestSchema,
+  OTAVersionResponseSchema,
+  OTACheckRequestSchema,
+  StoredVersionInfoSchema,
+  OTAMetricsSchema,
```

Comment:
Just one thing to point out:
We aren't using the schemas so we might as well keep them out of the export for the library.
Since these are `zod` schemas it can lead to version mismatching with someone who is using `zod` in their project.

Ideally we can work towards removing `zod` altogether.